### ISSUE TITLE
Stream deprecation, but no migration away notes.

### ISSUE THREAD

**Issue by SQLGuyChuck** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#5904]
If you plan on deprecating adding to Stream, will existing videos stay as is and for how long?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8935e913-806f-a92c-d09e-e67df4ffd1b4
* Version Independent ID: c7a41cde-66dd-d7cd-1316-42d20ed4841a
* Content: [Use OneDrive and SharePoint for meeting recordings - Microsoft Teams](https://docs.microsoft.com/en-us/MicrosoftTeams/tmr-meeting-recording-change)
* Content Source: [Teams/tmr-meeting-recording-change.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/tmr-meeting-recording-change.md)
* Service: **msteams**
* GitHub Login: @cichur
* Microsoft Alias: **v-cichur**

**Comment by cichur** [comment]
@SQLGuyChuck 
Thank you for your question. I got this confirmation from our SME. I also added your question to the document. 

Stream as a platform won't be deprecated in the near future. The videos that currently live in Stream will stay there until we start migrating. Upon migration, those videos will be migrated to ODSP as well. Check [here](https://docs.microsoft.com/stream/migrate-from-office-365) for more information.
